src/_check.sh

```diff
@@ -33,7 +34,7 @@ function mod_check() {
 function check_for_maintenance_mode() {
   title "Checking Maintenance Mode"
   IN_MAINTENANCE="null"
-  IN_MAINTENANCE=$(curl -ks https://${CLIENT_DOMAIN_NAME}/api/v2/health/full | jq .data.inMaintenanceMode)
+  IN_MAINTENANCE=$(curl -ks https://${CLIENT_DOMAIN_NAME}/api/v2/health/full | jq .data.inMaintenanceMode) || IN_MAINTENANCE="null"
```

Comment:
I think that below could be simplified to `info "Maintenance Mode: $IN_MAINTENANCE"` + 
```suggestion
  IN_MAINTENANCE=$(curl -ks https://${CLIENT_DOMAIN_NAME}/api/v2/health/full | jq .data.inMaintenanceMode) || IN_MAINTENANCE="unknown"
```

Reply:
fewer lines of code and less logic to thing about